### PR TITLE
Python arch specific patch

### PR DESCRIPTION
**Pull request by barry-ravichandran**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by barry-ravichandran** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,21 +10,22 @@ source:
...
   run:
-    - python >=3.8.1,<4.0.0
-    - numpy >=1.23.1
+    - python
+    - numpy >=1.23.1<1.26  # [py<312]
+    - numpy >=1.26  # [py<312]
     - opencv >=4.6
     - pillow >=10.2.0
-    - scikit-image >=0.21
+    - scikit-image >=0.21<0.22  # [py<312]
```
Comment (vicentebolea) [inline recipe/meta.yaml:27]: missing comman here and at 23

**Comment by barry-ravichandran** [comment]
@conda-forge-admin, please rerender

**Comment by barry-ravichandran** [comment]
@conda-forge-admin, please rerender

**Comment by vicentebolea** [comment]
@conda-forge-admin, please rerender

**Comment by vicentebolea** [comment]
a rebase/push -f might trigger the CI again to have the changes of pybsm

**Pull request merged**